client/index.js

```diff
@@ -31,6 +33,17 @@ import OverviewPage from 'overview';
 import DocumentsPage from 'documents';
 import OnboardingPage from 'onboarding';
 import { getTasks } from 'overview/task-list/tasks';
+import packageJson from '../package.json';
+
+// Initialise Sentry early on for error tracking.
+Sentry.init( {
+	dsn:
+		'https://3ff93a21f6a548ceb8ae22f2c9b8b9bd@o248881.ingest.sentry.io/6350586',
```

Comment:
Hard-coding like this might get numerous of false positives because you will get errors from the local environments. Can you maybe move this to the .env file and inject it via webpack?

Reply:
@zmaglica Good catch, I kept it like this for testing in dev environments, have now added an `enabled` param in the `init` method that sets it's value based on the `NODE_ENV`, thus sending only prod errors.

Reply:
@deepakpathania Proposed solution works too. Good job.